lua/telescope/_extensions/projects.lua

```diff
@@ -82,8 +83,13 @@ local function find_project_files(prompt_bufnr)
     hidden = config.options.show_hidden,
     mode = "insert",
   }
+  local _, ret, _ = utils.get_os_command_output({ "git", "rev-parse", "--is-inside-work-tree" })
```

Comment:
Should only run this command if both `cd_successful` is true and if `use_git_files` is true. No need to fork the process if either is false.

Reply:
Thank you for pointing that out. I updated the code as you said.